pywinauto/controls/HwndWrapper.py

```diff
@@ -1213,6 +1213,15 @@ def set_focus(self):
 
         return self
 
+    def has_keyboard_focus(self):
+        if self.get_focus() == self.handle:
+            return True
+        else:
+            return False
+
+    def set_keyboard_focus(self):
+        self.set_focus()
```

Comment:
How do you have this idea? Do you think I created this task just for making an aliases?